### DISCUSSION TITLE
Best Config

### DISCUSSION THREAD

**Discussion by stevefxp1** [Security-Onion-Solutions/securityonion#5368] [Unsupported Versions] [answered]
I have three NICs on my firewall, that connect to my internal core switch. Two of the three NICs are configured for VLANs and one NIC is passing a dedicated subnet.

I would like to sniff the traffic(ingress/egress) on each. Do I need a separate TAP for each NIC, as it seems I can only mirror to one destination port on my core switch. What’s the most effective way to handle this?

**Comment by dougburks** [comment, marked as answer]
Take a look at some of the inexpensive tap options here:
https://docs.securityonion.net/en/2.3/hardware.html#packets

**Comment by stevefxp1** [comment]
So I could use the Netgear switches as taps? On a 5 port switch I would use one port for the incoming connection, one port for the outgoing connection, and then one more port for Security Onion correct? As mentioned I have 3 interfaces from my firewall, so I would need 3 Netgear switches correct?

Lastly is there any opinion on using the Mikrotik switch instead of the Netgear?
